tests/transformers/test_modeling.py

```diff
@@ -12,13 +12,58 @@
 # See the License for the specific language governing permissions and
```

Comment:
let's rename this file to `test_model_utils.py` to be in sync with the `model_utils.py`